src/Service/Mailer/SymfonyMailer.php

```diff
@@ -64,4 +64,26 @@ public function sendInvitationToOrganization(string $email, string $token, strin
             ])
         );
     }
+
+    /**
+     * @param string[] $emails
+     * @param mixed[]  $result
+     */
+    public function sendScanResult(array $emails, string $packageName, string $packageId, string $organizationAlias, array $result): void
+    {
+        foreach ($emails as $email) {
```

Comment:
we can remove `foreach` and use `setBcc` to send one email to multiple receivers